packages/css/src/components/footer/README.md

```diff
@@ -2,16 +2,20 @@
 
 # Footer
 
-The footer is the closing section on every web page.
-Its content provides service information.
-A part is prescribed, and a part is filled based on the website’s objectives.
+Provides service information at the bottom of every page.
 
 ## Guidelines
 
-- The footer is mandatory, and there is one on every page.
+The Footer must be used on all websites for the City of Amsterdam.
+For applications, only its menu at the bottom can be sufficient.
```

Comment:
This is a little unclear imo. I think you mean the PageMenu here? Maybe we should move this to a story, so we can show how you do this.